frontend/src/app/general/header/header.component.html

```diff
@@ -21,16 +21,14 @@
     </div>
 
     <div class="hidden gap-2 xl:flex">
-      <div
-        *ngFor="let item of navBarService.navBarItems"
-        [routerLink]="item.routerLink"
-      >
+      <div *ngFor="let item of navBarService.navBarItems">
         <a
           mat-raised-button
           color="primary"
           *ngIf="userService.validateUserRole(item.requiredRole)"
           [attr.href]="item.href"
           [attr.target]="item.target"
+          [routerLink]="item.routerLink"
```

Comment:
The `href`-URLs (Grafana, Prometheus, Documentation) stop working with this change. This was also the reason why the routerLink was on the div element, when I remember it correctly.

In the worst case, we have to add conditional `a`-tags for routerLink & href.

Reply:
For some reason Angular won't let me conditionally bind href and target attributes when they exist and leave off routerLink (or vice versa), but because you'll always need to group same-page and new-page links to keep users sane I split them into two arrays with their own template logic. This is a little bit of duplication in the HTML but it's more explicit than it was previously (e.g. for each new page link add a new page icon instead of adding item.icon which was just always new page)

Reply:
Yes, the additional HTML is fine (I don't think we'll find a way around it anyway). 

I'd prefer one array over two arrays. You can then use a simple condition with ngIf to decide which HTML element to use. This would also allow to have a mixed order (same page, new page, same page, ...) in the future.

Reply:
I find this harder to read and maintain, but I already shared my general concerns with the nav, so done ✔️

Reply:
It will look better with the new control flow, but it's not there yet: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/a3903427-e4d3-40df-a8d5-8f1667415933)